packages/web-components/fast-foundation/src/design-token/design-token.ts

```diff
@@ -24,7 +24,16 @@ const defaultElement = document.body;
 export interface DesignToken<
     T extends string | number | boolean | BigInteger | null | Array<any> | symbol | {}
 > {
+    /**
+     * The name of the token
+     */
     readonly name: string;
+
+    /**
+     * A list of elements for which the DesignToken has a value set
+     */
+    readonly setFor: HTMLElement[];
```

Comment:
If anyone has better thoughts on a name for this I'm all ears.

Reply:
`appliedTo` ?

Reply:
yea that's better